INSTALL.md

```diff
@@ -163,6 +184,12 @@ Next, make a test environment:
 qiita-env make --no-load-ontologies
 ```
 
+Finally, redbiom rely on the REDBIOM_HOST environment variable to set the URL to query. By default is set to http://127.0.0.1:7379, which is the webdis default. For example you could:
```

Comment:
k, makes sense for default installations but it might be better to leave as is as the idea of these instructions is to have local installations for development.